packages/js/tracks/src/test/index.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Internal dependencies
+ */
+import { recordEvent } from '..';
+
+const eventName = 'my_event_name';
+const props = {
+	test: 'test value',
+};
+
+jest.mock( '../utils', () => ( {
+	isDevelopmentMode: false,
+} ) );
```

Comment:
I had to mock this because when I'm running the test on my local `process.env.NODE_ENV` is `test` but in GH it's `development`.